include/glow/Backends/DeviceManager.h

```diff
@@ -152,6 +153,35 @@ class DeviceManager {
   /// \returns the DeviceInfo for this device containing peak limits for
   /// compute and bandwidths (used in partitioning).
   virtual DeviceInfo getDeviceInfo() const { return DeviceInfo(); }
+
+  /// Copies the contents of \p tensor from the host to the \p location
+  /// address on this device. Updates the tensor residency info.
+  virtual bool transferToDevice(
```

Comment:
Do we need to return a bool here or can this be void? It seems with the callback returning the error the bool is not needed.

Reply:
void should be fine now, good point